processor/k8sprocessor/options.go

```diff
@@ -68,30 +71,30 @@ func WithExtractMetadata(fields ...string) Option {
 	return func(p *kubernetesprocessor) error {
 		if len(fields) == 0 {
 			fields = []string{
-				metdataNamespace,
-				metadataPodName,
-				metadataPodUID,
-				metadataStartTime,
-				metadataDeployment,
-				metadataCluster,
-				metadataNode,
+				conventions.AttributeK8sNamespace,
+				conventions.AttributeK8sPod,
+				conventions.AttributeK8sPodUID,
+				metadataPodStartTime,
+				conventions.AttributeK8sDeployment,
+				conventions.AttributeK8sCluster,
+				conventions.AttributeK8sNodeName,
 			}
 		}
 		for _, field := range fields {
 			switch field {
-			case metdataNamespace:
+			case metdataNamespace, conventions.AttributeK8sNamespace:
```

Comment:
Please leave a comment explaining why the old conventions are still used, and maybe a tentative date when these can be removed if possible.